conda_build/environ.py

```diff
@@ -385,6 +385,8 @@ def unix_vars(prefix):
 
 def osx_vars(compiler_vars, config):
     OSX_ARCH = 'i386' if config.arch == 32 else 'x86_64'
+    MACOSX_DEPLOYMENT_TARGET = os.environ.get('MACOSX_DEPLOYMENT_TARGET', '10.7')
+
     compiler_vars['CFLAGS'] += ' -arch {0}'.format(OSX_ARCH)
     compiler_vars['CXXFLAGS'] += ' -arch {0}'.format(OSX_ARCH)
```

Comment:
I'm fine to merge this as is.  If you want to add that later, let's discuss more at that time.